packages/appsync-modelgen-plugin/src/configs/dart-config.ts

```diff
@@ -1,11 +1,16 @@
 export const LOADER_CLASS_NAME = 'ModelProvider';
 export const BASE_IMPORT_PACKAGES = [
   'package:flutter/foundation.dart',
-  'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart'
+  'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart',
```

Comment:
Formatting changes here and at the end of the file were added automatically in a post commit hook